README.md

```diff
@@ -55,6 +55,24 @@ wasmer run examples/sqlite.wasm
 wasmer run examples/nginx/nginx.wasm -- -p examples/nginx -c nginx.conf
 ```
 
+#### With WAPM
+
+Installing Wasmer through `get.wasmer.io` includes
```

Comment:
`get.wasmer.io` may be a bit misleading. Going to that link just shows the shell script. May I suggest `wasmer.io` instead?